src/YieldDistributor.sol

```diff
@@ -227,6 +232,19 @@ contract YieldDistributor is IYieldDistributor, OwnableUpgradeable {
         _castVote(msg.sender, _points, _currentVotingPower);
     }
 
+    /**
+     * @notice Cast votes for the distribution of $BREAD yield with multipliers
+     * @param _points List of points as integers for each project
+     * @param _multiplierIndices List of indices of multipliers to use for each project
+     */
+    function castVoteWithMultipliers(uint256[] calldata _points, uint256[] calldata _multiplierIndices) public {
+        uint256 _currentVotingPower = getCurrentVotingPower(msg.sender);
+        if (_currentVotingPower < minRequiredVotingPower) revert BelowMinRequiredVotingPower();
```

Comment:
Are we sure we want to check this requirement before multipliers are applied??

Reply:
trueeeee

Reply:
addressed with 586f8b0